docs/manual/java/releases/Migration15.md

```diff
@@ -122,12 +122,9 @@ When opting in to Akka Cluster Bootstrapping as a mechanism for Cluster formatio
 
 ## Upgrading a production system
 
-TODO
-
-### Rolling upgrade
-
-TODO
+There are no changes affecting a Production Upgrade. If you are running a Lagom 1.4 cluster you can perform a rolling upgrade, just make sure you are using thhe latest version on the 1.4.x series and, from that, you migrate to the latest version available of the 1.5.x series.
```

Comment:
there is a typo in `thhe`

Reply:
yeah, @ignasi35 has a failing Mac keyboard. That typo appears from time to time. Or should I say, from keystroke to keystroke?

Reply:
```suggestion
There are no changes affecting a Production Upgrade. If you are running a Lagom 1.4 cluster you can perform a rolling upgrade, just make sure you are using the latest version on the 1.4.x series and, from that, you migrate to the latest version available of the 1.5.x series.
```

Reply:
I started a crowdfunding campaign to buy a keyboard: www.buybuybuy-me-a-keyboard.com